src/ucp/rndv/rndv.c

```diff
@@ -1808,6 +1806,43 @@ static ucs_status_t ucp_rndv_send_start_put_pipeline(ucp_request_t *sreq,
     return UCS_OK;
 }
 
+static UCS_F_ALWAYS_INLINE ucp_request_t*
+ucp_rndv_rma_get_zcopy_req(ucp_request_t *req)
```

Comment:
is this function still needed?

Reply:
good catch, the function is not needed anymore - removed